Imports `@ucast/mongo2js` without being defined in `package.json`

Noticed this while trying to add this project to a Yarn 3 PnP project. See:

https://github.com/getjerry/nest-casl/blob/a3af8364f8da96563f6ae16d9f487725c2631355/src/proxies/conditions.proxy.ts#L3

Seems like that dependency should be added to `package.json`.

To clarify, the project fails to start because of this:

```
14:44:29 - Starting compilation in watch mode...

../../.yarn/__virtual__/nest-casl-virtual-9ca4a67998/0/cache/nest-casl-npm-1.4.1-eb65e72b31-d1dd270042.zip/node_modules/nest-casl/dist/proxies/conditions.proxy.d.ts(2,27): error TS2307: Cannot find module '@ucast/mongo2js' or its corresponding type declarations.

14:44:52 - Found 1 error. Watching for file changes.
There was a problem with the build!
```

See: https://yarnpkg.com/getting-started/migration/#fix-dependencies-with-packageextensions

--- a/package.json
+++ b/package.json
@@ -38,6 +38,7 @@
     "@casl/ability": "^5.4.3",
     "@ucast/core": "^1.10.1",
     "@ucast/mongo": "^2.4.2",
+    "@ucast/mongo2js": "^1.3.3",
     "@ucast/sql": "^1.0.0-alpha.1"
   },
   "peerDependencies": {

